lib/elixometer.ex

```diff
@@ -281,6 +281,9 @@ defmodule Elixometer do
   @doc """
   Updates a counter metric. If the metric doesn't exist, the metric is created
   and the metric is subscribed to the default reporter.
+
+  If the value of the `:reset_seconds` option is greater than zero, the counter will be reset
+  automatically at the specified interval.
   """
   def update_counter(name, delta, opts \\ [reset_seconds: 60]) when is_bitstring(name) do
```

Comment:
I think making `reset_seconds` nil by default (no reset) makes more sense. I don't remember why I made the default 60;